docs/current/guides/snippets/build-test-publish-php/dagger.php

```diff
@@ -0,0 +1,277 @@
+<?php
+// include auto-loader
+include 'vendor/autoload.php';
+
+use GraphQL\Client;
+
+class DaggerPipeline {
+
+  private $client;
+
+  // constructor
+  public function __construct() {
+    // initialize client with
+    // endpoint from environment
+    $sessionPort = getenv('DAGGER_SESSION_PORT') or throw new Exception("DAGGER_SESSION_PORT doesn't exist");
```

Comment:
I prefer to make these instructional so that the user knows what they need to do to fix the issue, e.g. `DAGGER_SESSION_PORT environment variable must be set`

Reply:
Done